src/arrayMethodJoin.js

```diff
@@ -3,9 +3,24 @@
 /**
  * Implement method join
  */
+
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let string = '';
+
+    for (let i = 0; i < this.length; i++) {
+      if (this[i] == null) {
+        string += ``;
+      } else {
```

Comment:
```suggestion
      if (this[i] !== null) {
```
Can't you just do nothing?
Also, always use === instead of ==
![image](https://user-images.githubusercontent.com/96409563/213909207-1fb9065d-3089-47a3-84ef-07f36f305a21.png)

Reply:
I have a question though. I used
(value != null) 
as a shorthand for
(value !== null && value !== undefined)
Is it wrong?